click_odoo_contrib/initdb.py

```diff
@@ -415,7 +420,7 @@ def main(
     """
     if new_database:
         check_dbname(new_database)
-    if unless_exists and db_exists(new_database):
+    if unless_exists and db_initialized(new_database):
```

Comment:
This changes the semantics of `--unless-exists` and it looks like its breaking a test indeed.